yarn-project/circuits.js/src/scripts/constants.in.ts

```diff
@@ -322,6 +322,9 @@ function evaluateExpressions(expressions: [string, string][]): { [key: string]:
   const prelude = expressions
     .map(([name, rhs]) => {
       const guardedRhs = rhs
+        // Remove 'as u8' and 'as u32' castings
+        .replaceAll(' as u8', '')
+        .replaceAll(' as u32', '')
```

Comment:
With the bit shifts I had to introduce castings and this gets rid of them in TS.